### PR TITLE
LeVerne Johnson

### PR DESCRIPTION
**Pull request by teko000**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [x] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [x] Effective use of box model: 15 points
- [x] Proper use of documentation (commenting on code): 15 points
- [x] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 25 points

**Checkpoint 2**
- [x] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [x] Effective use of Bootstrap Grid: 15 points
- [x] Proper use of documentation (commenting on code): 15 points
- [x] Site is hosted on github: 15 points
- [x] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 25 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 01week/resume/index.html [hunk 1]
```diff
@@ -1,49 +1,98 @@
...
+  <title>LeVerne Johnson's Resume</title>
+  <link rel="stylesheet" href="web-workbook/01week/resume/">
 </head>
 
 <body>
   <header>
-    LeVerne Johnson<br> 1313 Mockingbird Lane<br> Austin, Tx 78634<br>
-    <a href="mailto:teko0010@yahoo.com?Subject=Hello" target="_top">teko0010@yahoo.com</a>
+    LeVerne Johnson<br> 1313 Mockingbird Lane<br> Austin, Tx 78705<br>
+    <a href="mailto:teko0010@yahoo.com?Subject=Resume" target="_top">teko0010@yahoo.com</a>
```
Comment (e-zambrano) [inline 01week/resume/index.html]: Nice, I like know you found out how to also include a subject for email sending

**Comment by e-zambrano** [comment]
LeVerne,
Good work on structuring your resume using semantic HTML elements
I'd like you to bring more questions during class!

**Comment by teko000** [comment]
Ok,
  Will do.

File: 02week/bio/css/style.css [hunk 2]
```diff
@@ -1,48 +1,46 @@
-a:hover{
-  color:white;
+a:hover {
+  color: white;
 }
 
 .img1 {
```
Comment (e-zambrano) [inline 02week/bio/css/style.css]: Your classes `.img1` and `.img2` are too generic. Make sure that you name your classes more semantic like `.profile-picture` or `.state-logo`

**Comment by e-zambrano** [comment]
Good Job, LeVerne. Nice work by adding shadow in your text

File: 01week/resume/index.html [hunk 3]
```diff
@@ -1,15 +1,18 @@
+<!doctype html>
 <html>
+<link href="css/style.css" rel="stylesheet" type="text/css" />
 
 <head>
   <title>LeVerne Johnson's Resume</title>
-  <link rel="stylesheet" href="style.css">
+  <link href="css/style.css" rel="stylesheet" type="text/css" />
```
Comment (e-zambrano) [inline 01week/resume/index.html]: This `link` element is already added in your file in line 3.
Remove it

File: 02week/bio/index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html?
```
Comment (e-zambrano) [inline 02week/bio/index.html]: It ends with a `>` not with a `?`

File: 02week/bio/index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        Dicta complectitur pro teus phaedrum duo. Cu doming ceteros intellegat qui, has appareat contentiones ea. Eu equidem commune verterem eam, an munere salutandi appellantur sed. Sea nobis putant consectetuer ad, mea habeo eripuit ex. Sint dolore postulant in sit, cu esse intellegebat qui.
+        Sit cu zljlkjlkjri-l tibique ponderum. Etfalkasjd;lfkja eoirja ;oeirj a;oijfape9riwpe49irpawowpoiwpoifas-asfdaer-afear-asefd-ere-efasdf-eera4er-weef sea munere similique disputationi, at cum eligendi forensibus. Vis solet electram dissentias no, dicat quaerendum nec et. Per et ipsum putant concludaturque, sit ut nostrum deseruisse. Et ius omnis zril, mei ad singulis mediocrem dignissim, libris euripidis repudiandae sea ut.
+        Ridens adversarium ea vel, odio maiorum voluptua no ius. Ius inermis accusamus maiestatis ei, no disputando consectetuer vis. Eu tota lobortis nam, solum praesent abhorreant his ut, mei cu illum corrumpit patrioque. Eu dicam scriptorem vis, diam equidem neglegentur te quo. Vix in tincidunt elaboraret, probo adversarium ad nec.
+        Save-The-Best-For-Last.
+      </p>
+    </section>
+
+    <section class ="center2">
```
Comment (e-zambrano) [inline 02week/bio/index.html]: Look for better class names

**Comment by e-zambrano** [comment]
LeVerne,
I see you tried to add a faq and about page to the blog project. When I click on those links a get an 404 error, which means that the reference you added in your `<a>` element is incorrect.
Overall, make sure that you are creating classes with more meaningful names, avoid `class1`, `class2`, etc.

**Comment by teko000** [comment]
Thanks,
  I meant to take that out.

**Comment by teko000** [comment]
Thanks,
  That was a typo I make often using the laptop keyboard.

File: 04week/nasa/index.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="css/style.css" rel="stylesheet" type="text/css" />
+    <title>Nasa</title>
+  </head>
+  <body>
+    <div class="galaxy">
+      <p>
```
Comment (e-zambrano) [inline 04week/nasa/index.html:9]: Forgot to style this element - I can barely see it.

File: 04week/nasa/css/style.css [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  align-items: baseline;
+  flex-direction: row;
+  background-color: #E6E6E6;
+  margin-top: 20px;
+  margin-left: 2px;
+}
+
+.office-info{
+width:270px;
+  height:320px;
```
Comment (e-zambrano) [inline 04week/nasa/css/style.css]: You already provided the height to the parent (.info) and this is making it bleed out of it's parent.
image
Same goes for `.contact-us`

**Comment by e-zambrano** [comment]
Great work on the use of flexbox. On your "tomorrows-technology" project you are missing the tablet media query.

**Comment by e-zambrano** [comment]
I don't know how you manage to have to folder `05week` and 05week0` with the same content. Please make sure that you only carry with a single folder that week

**Comment by e-zambrano** [comment]
@teko000, you will need to revisit the file `08week/Project/js/script.js` and compare it with the solution.
You are writing the same `$(document).ready()` three times. It should have been once.

`window.onload` and `$(document).ready()` represent the same as far as when the code it's going to get executed. 

The `click all` it's not triggering the other two buttons

**Pull request closed without merging**